src/org/pcic/util/Utils.groovy

```diff
@@ -71,4 +71,33 @@ class Utils implements Serializable {
     boolean isPublishable(String branchName, ArrayList tags) {
         return (branchName == 'master' && !tags.isEmpty())
     }
+
+    /**
+     * Installs packages using apt-get
+     *
+     * In some cases before we can install a package the package list must be
+     * updated.  Updating the list is a costly operation timewise and thus we'd
+     * like to avoid running it unnecessarily.  Before running the installation
+     * we check against a list of packages known to require an update.
+     *
+     * @param packages list of packages to install
+     */
+    void installAptPackages(ArrayList packages) {
+        ArrayList requireUpdate = ['git'].intersect(packages)
```

Comment:
`['git']` is a magic "number" here. Your comment makes it clear what it is for, but nonetheless it should at least be declared as a named constant. Possibly it should be in a configuration file, but that may be a step to far at this point.

Reply:
Should it be an attribute for the class?

Reply:
That's certainly reasonable. You could 'hoist' it higher (into a config file, e.g., to take it entirely out of code), but it probably isn't worth the effort at this point.

Reply:
I've pulled out the variable into a "named constant" that exists as an attribute to a class.  The class is in `util/Global.groovy` which allows for it to be imported from the `org.pcic.util` package.  Do you feel like this is an adequate solution? If not I can look to implement a configuration file.

Reply:
That seems perfectly fine.